packages/lib/src/toolbar/controls/ExportMenu.tsx

```diff
@@ -8,11 +8,12 @@ import styles from './Selector/Selector.module.css';
 
 interface Props {
   entries: ExportEntryProps[];
-  isSlice: boolean;
+  isSlice?: boolean;
+  align?: 'center' | 'left' | 'right';
```

Comment:
Allow aligning the export menu to the right of the "Export" button to fix overflow issues in the Raw and Matrix toolbars.